spring-jdbc/documentation.md

```diff
@@ -0,0 +1,188 @@
+# The Crate JDBC Sample Application
+
+**Note:** The application is written in Java 11.
```

Comment:
I would go with Java 13.

Reply:
I've chosen Java 11 since it's the newest LTS version of Java